File: Data Structures & Algorithms/two-integer-sum-ii/submission-4.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {

        int l=0,r=numbers.size()-1;

        while(l<r){
            if(numbers[l]+numbers[r]>target){
                r--;
            }
            else if(numbers[l]+numbers[r]<target){
                l++;
            }
            else{
                return {numbers[l],numbers[r]};
            }
        }
        return {};
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-4.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        l=0

        while l<len(nums):
            for i in range(l+1,len(nums)):
                if target-nums[l]==nums[i]:
                    return [l,i]
            l+=1
        
        